docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.2"
 services:
   parity:
-    image: makerdao/testchain-pymaker:unit-testing
+    image: makerdao/testchain-pymaker:unit-testing-1.1.0
```

Comment:
No, we don't want to introduce an environment variable for this.  It should be specified here, in the testchain deployment, and nowhere else.